developer_manual/client_apis/WebDAV/basic.rst

```diff
@@ -78,28 +100,184 @@ You can request additional properties by sending a request body with the :code:`
+|                               |                                                  | | ``9`` = Remote group                                                   |
+|                               |                                                  | | ``10`` = Talk conversation                                             |
+|                               |                                                  | | ``12`` = Deck                                                          |
+|                               |                                                  | | ``15`` = Science mesh                                                  |
++-------------------------------+--------------------------------------------------+--------------------------------------------------------------------------+
+| <ocs:share-permissions />     | | The permissions that the                       | | ``1`` = Read                                                           |
+|                               | | user has over the share.                       | | ``2`` = Update                                                         |
+|                               |                                                  | | ``4`` = Create                                                         |
+|                               |                                                  | | ``8`` = Delete                                                         |
+|                               |                                                  | | ``16`` = Share                                                         |
+|                               |                                                  | | ``31`` = All                                                           |
++-------------------------------+--------------------------------------------------+--------------------------------------------------------------------------+
+| <ocm:share-permissions />     | | The permissions that the                       | ``share``, ``read``, ``write``                                           |
+|                               | | user has over the share.                       |                                                                          |
++-------------------------------+--------------------------------------------------+--------------------------------------------------------------------------+
+| <ocm:share-attributes />      | User set attributes.                             | ``[{ "scope" => <string>, "key" => <string>, "enabled" => <bool> }]``    |
++-------------------------------+--------------------------------------------------+--------------------------------------------------------------------------+
+| <nc:sharees />                | The list of share recipient.                     | ``{ id, display-name, share type }``                                    |
```

Comment:
https://github.com/nextcloud/server/blob/278f6413c9ee6d5c011c59a81d2f8e24a86dfbac/apps/dav/lib/Connector/Sabre/ShareeList.php uses xml for the value